### PR TITLE
Single filter northern ecliptic spur

### PR DESCRIPTION
**Pull request by mschwamb**
proposal for an opsim with the northern ecliptic spur observed in a single filter

### CODE REVIEW

**Comment by mschwamb** [comment]
@drphilmarshall I figure this should be looked at rather than me directly merging this in. I wasn't sure who to put, so I put you. If you're not the right person, do let me know who to assign. Thanks

**Comment by rhiannonlynne** [comment]
@drphilmarshall did travis just die?

File: opsim/Proposal_single_filter_northern_ecliptic_spur.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+A Proposed OpSim Experiment.
+
+*Proposer(s): Meg Schwamb*
+
+*Issue Thread: NA*
+
+
+## Proposed Changes to the Observing Strategy
+
+Observe the Northern Ecliptic Spur region in a single filter (either r or g) only
```
Comment (drphilmarshall) [inline opsim/Proposal_single_filter_northern_ecliptic_spur.md]: Might be worth spelling out how this relates to the usual Northern Spur proposal (specifying which simulation - minion1016? - you are proposing a variation of.  It sounds as though you are suggesting to reduce the overall number of visits in this region, but it'd be good to be specific.

File: opsim/Proposal_single_filter_northern_ecliptic_spur.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+
+## Science Motivation
+
+The Northern Ecliptic Spur is a ~4,000 degree proposed minisurvey along the ecliptic in the northern hemisphere that has larger airmasses than the main wide survey observations. It is aimed to cover the large swath of the ecliptic not covered in the wide survey, important for Solar System science. The Solar System Science Collaboration (SSSC) would like to study the possible impacts to our top priority science cases if the Northern Ecliptic Spur is only observable in a single filter due to other cadence restrictions. The results from this opsim run would better inform candence decisions and help the better design this minisurvey and other future proposed deep drilling fields.
```
Comment (drphilmarshall) [inline opsim/Proposal_single_filter_northern_ecliptic_spur.md]: "only observed"?

**Comment by drphilmarshall** [comment]
@rhiannonlynne Looks like it - the texlive install no longer works....

File: opsim/Proposal_single_filter_northern_ecliptic_spur.md (hunk 1 above)
Comment (mschwamb) [reply opsim/Proposal_single_filter_northern_ecliptic_spur.md]: There are improvements coming to opsim and I presume new simulations, so I'm not sure it's worth specifying the specific flavor.  I need to know how any changes to the North Ecliptic Spur will impact Solar System science especially once rolling cadence is implemented. So I'm saying to only observe the northern ecliptic spur fields in  a single filter (r or g).

File: opsim/Proposal_single_filter_northern_ecliptic_spur.md (hunk 2 above)
Comment (mschwamb) [reply opsim/Proposal_single_filter_northern_ecliptic_spur.md]: sure I can go with that word choice.

File: opsim/Proposal_single_filter_northern_ecliptic_spur.md (hunk 1 above)
Comment (drphilmarshall) [reply opsim/Proposal_single_filter_northern_ecliptic_spur.md]: OK cool - when @ivezic and his team circle round to your proposal, they'll ask for more detail about how you want the simulation set up. Until then, I'll merge this in.

**Pull request merged**